### PR TITLE
[EMCAL-524,EMCAL-526] Plots for different trigger classes

### PR DESCRIPTION
**Pull request by cterrevo**
Create and fill QA plots for Calib and Physics Triggers in Raw and Digit
tasks

### CODE REVIEW

File: Modules/EMCAL/src/DigitsQcTask.cxx [hunk 1]
```diff
@@ -142,21 +179,34 @@ void DigitsQcTask::monitorData(o2::framework::ProcessingContext& ctx)
     //gsl::span<const o2::emcal::Digit> eventdigits(digitcontainer.data() + trg.getFirstEntry(), trg.getNumberOfObjects());
     gsl::span<const o2::emcal::Cell> eventdigits(digitcontainer.data() + trg.getFirstEntry(), trg.getNumberOfObjects());
 
+    //trigger type
+    auto triggertype = trg.getTriggerBits();
+    bool isPhysTrigger = triggertype & o2::trigger::PhT, isCalibTrigger = triggertype & o2::trigger::Cal;
+    std::string trgClass;
+    if (isPhysTrigger)
```
Comment (Barthelemy) [inline Modules/EMCAL/src/DigitsQcTask.cxx:186]: we recommend to always add curvy braces.

**Review by Barthelemy — COMMENTED** [review]
Ok, just 1 comment.

**Review by Barthelemy — APPROVED** [review]

**Comment by Barthelemy** [comment]
@ktf @TimoWilken I think that there is something wrong with the mac builder.

**Comment by TimoWilken** [comment]
@Barthelemy looking into it!

**Comment by TimoWilken** [comment]
@Barthelemy I think I've found the issue - the build timed out, very probably because of a corrupt git repo for some dependencies. I've fixed it for now and I'll have a look at how to prevent this in future. In the meantime, I've marked the failed builds as pending so they run again.

File: Modules/EMCAL/src/DigitsQcTask.cxx (hunk 1 above)
Comment (mfasDa) [reply Modules/EMCAL/src/DigitsQcTask.cxx:186]: OK, we take this as recommendation :)

**Comment by mfasDa** [comment]
To make clear: We need a change in O2, because we need to propagate the trigger bits in one of our headers in order to distinguish different triggers. The change is merged into dev there.

We tested the change with the data we just collected the past few days with the expected setup for the VS test and we are ready with this. It is important for us to get this commit merged before the MW1.

**Comment by Barthelemy** [comment]
@mfasDa There are real errors in the o2 build that must be fixed, otherwise it would break everything. 
For the mac build, I am not sure why it fails if the change in O2 is there. Or is it just an old build ? 

I am not sure that it is possible to have it in time for MW1 to be honest. We need to have the CI pass here, create a new version of QC before the FLP suite is locked.

**Comment by mfasDa** [comment]
Should be closed as the PR is split into a PR for the raw task https://github.com/AliceO2Group/QualityControl/pull/590 which does not rely on changes in O2 and should be handled with priority, and a commit for the cell task https://github.com/AliceO2Group/QualityControl/pull/589 which relies on changes in O2.

**Pull request closed without merging**